packages/app-spec-squad-chess/src/FormatSelector.js

```diff
@@ -0,0 +1,20 @@
+import m from "mithril"
+import state from "./state.js"
+
+const FormatSelector = {
+  view: () => {
+    return m(
+      '#format-selector',
+      m('h3', 'Available Formats'),
+      state.rawFormats.map((rawFormat, index) => {
+        const url = String(window.location)
```

Comment:
Let's use the javascript `URL` api here. 

More info here https://developer.mozilla.org/en-US/docs/Web/API/URL

but it would look something like 

```
url = new URL(window.location)
url.search = `format=${index}`
console.log(url.href)
```